lib/github-glue.ts

```diff
@@ -464,10 +464,6 @@ export class GitHubGlue {
             username: login,
         });
 
-        if (!response.data.name) {
```

Comment:
Sorry.  Does the return need to be `name: response.data.name || "",`?  The `name` changed from string to string|null.  Returning an empty string vs changing the interface and handling other upstream changes (ie `lib/gitgitgadget.ts:284:44`) seems simpler.

Reply:
Hmmm.  The upstream error is on using `userInfo.name`.  That should probably be changed to `pr.author` if null user names are allowed.

Reply:
Null user names are rejected because a name is required on whose behalf GitGitGadget sends the mails (or at least the cover letter). However, we do support `/allow`ing before a name is set (but `/submit`ing will still fail, with the error message "Error: Could not determine full name of [name]", see [here](https://github.com/git/git/pull/943#issuecomment-756867961)).

> Does the return need to be `name: response.data.name || "",`?

Good point, I made it so.

Reply:
> Hmmm. The upstream error is on using `userInfo.name`. That should probably be changed to `pr.author` if null user names are allowed. Also means the interface could be changed to allow null.

Just noting for future clarity this comment is incorrect since the two values are not equivalent.